### ISSUE TITLE
when sft  with xverse-13b model,  eval loss  always nan

### ISSUE THREAD

**Issue by liwenju0** [shibing624/MedicalGPT#174]
### Describe the bug
Please provide a clear and concise description of what the bug is. If applicable, add screenshots to help explain your problem, especially for visualization related problems.
训练阶段：
sft
训练数据：
shibing624-sharegpt_gpt4
训练参数：
CUDA_VISIBLE_DEVICES=0,1 torchrun --nproc_per_node 2 supervised_finetuning.py \
    --model_type auto \
    --model_name_or_path /home/deepctrl/liwenju/origin_models/XVERSE-13B \
    --train_file_dir ./data/finetune \
    --max_source_length 1024 --max_target_length 1024 \
    --num_train_epochs 3 \
    --per_device_train_batch_size 6 \
    --per_device_eval_batch_size 4 \
    --do_train \
    --do_eval \
    --use_peft True \
    --fp16 \
    --max_train_samples -1 \
    --max_eval_samples -1 \
    --learning_rate 1e-4 \
    --warmup_ratio 0.05 \
    --weight_decay 0.05 \
    --logging_strategy steps \
    --logging_steps 10 \
    --eval_steps 50 \
    --evaluation_strategy steps \
    --save_steps 200 \
    --save_strategy steps \
    --save_total_limit 10 \
    --gradient_accumulation_steps 1 \
    --preprocessing_num_workers 4 \
    --output_dir outputs-sft-xverse-v1 \
    --overwrite_output_dir \
    --ddp_timeout 30000 \
    --logging_first_step True \
    --target_modules all \
    --lora_rank 64 \
    --lora_alpha 128 \
    --lora_dropout 0.05 \
    --torch_dtype bfloat16 \
    --device_map auto \
    --report_to tensorboard \
    --ddp_find_unused_parameters False \
    --gradient_checkpointing True --qlora True --cache_dir ./cache

gpu： 
2 个A800 80G

训练 loss 不收敛：
<img width="437" alt="image" src="https://github.com/shibing624/MedicalGPT/assets/16049564/b8039d5b-deb7-49e2-b906-fbddda0646f6">
eval loss 为 nan

**Comment by shibing624** [comment]
1.手动指定下验证集；
2.base model 试试llama, xverse 模型没有验证过
